bedrock/firefox/templates/firefox/home/index-quantum.html

```diff
@@ -192,69 +150,82 @@ <h2 class="section-title">{{ _('2x Faster') }}</h2>
+          image_url='firefox/home/quantum/desktop-smooth.svg',
+          aspect_ratio='mzp-has-aspect-1-1',
+          class='mzp-l-card-feature-right-half mzp-t-firefox t-smooth',
+          link_url='https://blog.mozilla.org/firefox/quantum-performance-test/',
+          link_cta=_('Learn More')
+        ) %}
+        <p>
+          {% trans url='https://blog.mozilla.org/firefox/de/firefox-quantum-superschnell-und-schont-den-speicher/' if LANG == 'de' else 'https://blog.mozilla.org/firefox/quantum-performance-test/' %}
+            <a href="{{ url }}">Less memory usage</a> means more space for your computer to keep running smoothly.
+            Your other programs will thank you.
+          {% endtrans %}
+        </p>
+        {% endcall %}
       </div>
     </div>
-  </section>
-
-  <section class="page-section secondary-download-section" data-scroll-tracking="Footer download CTA">
-    <div class="content">
-    {% if LANG.startswith('en') %}
-      <h2><a href="{{ url('firefox.new') }}">The new <strong>Firefox</strong></a></h2>
-    {% else %}
-      <h2><a href="{{ url('firefox.new') }}">{{ _('The New Firefox') }}</a></h2>
-    {% endif %}
-      <p>{{ _('Meet Firefox Quantum.') }}</p>
-      <p>{{ _('Fast for good.') }}</p>
-      {{ download_firefox(dom_id='footer-download', alt_copy=_('Download now'), download_location='other') }}
+    <div class="mzp-l-content">
+      <div data-scroll-tracking="Switch from Chrome to Firefox in just a few minutes">
+        {% if l10n_has_tag('switch_cta_021218') %}
+          {% set switch_title = _('Switch from Chrome to Firefox in just a few minutes') %}
+          {% set switch_desc =_('Switching to Firefox is fast, easy and risk-free. Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.') %}
+        {% else %}
+          {% set switch_title = _('Moving browsers? We got you.') %}
+          {% set switch_desc =_('Instantly import your online info and favorites from any other browser.') %}
+        {% endif %}
+        {% call feature_card(
+          title=switch_title,
+          image_url='firefox/home/quantum/desktop-change.png',
+          include_highres_image=True,
+          aspect_ratio='mzp-has-aspect-1-1',
+          class='mzp-l-card-feature-left-half mzp-t-firefox t-switch',
+          link_url=url('firefox.switch'),
+          link_cta=_('Learn More')
+          ) %}
+          <p>{{ switch_desc }}</p>
+        {% endcall %}
+      </div>
     </div>
   </section>
-
+  <div data-scroll-tracking="Footer download CTA">
+    {% call call_out_compact(
+      title=_('Fast for good.'),
+      class='mzp-t-firefox mzp-t-product-firefox mzp-t-dark t-you'
+      ) %}
+      <div class="download-firefox">
+        {{ download_firefox(dom_id='footer-download', alt_copy=_('Download now'), download_location='other') }}
+      </div>
+    {% endcall %}
+  </div>
 </main>
 {% endblock %}
-
-{% block js %}
-  {{ js_bundle('firefox-quantum') }}
```

Comment:
I notice the nav on this old page is no longer sticky since this bundle was removed. I'm easy with removing it, but if it's intentional then we can also remove this bundle from static files?

Reply:
It was intentional so I'll remove the bundle 👍

Reply:
Er, making it not sticky was intentional, it should still have the rest of the JS 😞